libfuse/dir.go

```diff
@@ -469,6 +470,18 @@ func (d *Dir) Lookup(ctx context.Context, req *fuse.LookupRequest, resp *fuse.Lo
 		return node, nil
 	}
 
+	if strings.HasPrefix(req.Name, libfs.FileInfoPrefix) {
```

Comment:
Both here and in libdokan, it would be a little clearer to either have a comment above here explaining what this is doing, or to move it out into a helper function, so it doesn't confuse readers of the Lookup function too much.